bundles/org.openhab.binding.ring/src/main/java/org/openhab/binding/ring/internal/config/AccountConfiguration.java

```diff
@@ -28,4 +28,5 @@ public class AccountConfiguration {
     public int videoRetentionCount;
     public String videoStoragePath = "";
     public int refreshInterval;
+    public boolean limitToOwner = false;
```

Comment:
`false` is the default value: 

```suggestion
    public boolean limitToOwner;
```